### DISCUSSION TITLE
Island Travel Art Request

### DISCUSSION THREAD

**Discussion by Rofrtd** [sunflower-land/sunflower-land#1514] [Art]
This Discussion is a request for pixel art to be used in game for the island travel mechanic that will be used in Land Expansion.

As Land Expansion will contain many islands in which players will be able to travel to, we are looking for some art from the community. 

Please submit the following items as separate files: 
1. A Pirate Goblin with a pirate hat and an eye patch.
2. A boat and/or raft

We're also accepting submissions for the island icons (to show on the list of available islands to travel to), currently we have 4 Islands:

* Main Island (player land)
* Helios (where we have the Grub, Potion and Magic Seeds shops)
* Goblin Retreat
* Snow Kingdom

#### Requirements

* Pirate size follows the current goblins sizing
* Fits game aesthetic and colours
* Original pieces of work

By contributing to the project you agree to the terms and conditions - https://docs.sunflower-land.com/support/terms-of-service.

#### Submissions

When submitting art, please include the following:

* Your farm ID (we will make sure the artists get some goodies 🏆 )
* A minified version of your art piece
* An example of the art used in the scene below

![195388032-fcb71daf-0ae2-4d23-aa2e-0b16ea294a36](https://user-images.githubusercontent.com/25412194/195458688-318d8d97-ddc4-4089-a002-5861976960fb.png)

#### Example
<img width="584" alt="Screen Shot 2022-10-13 at 12 43 27 am" src="https://user-images.githubusercontent.com/25412194/195359659-8f1c0726-c782-46df-8de8-916da0209559.png">

#### Deadline

We are so close to Land Expansion launch so we will be looking to add these items in on the 19th of October

Good luck farmers 🌻

**Comment by paluras** [comment]
![Pier](https://user-images.githubusercontent.com/98059798/195388032-fcb71daf-0ae2-4d23-aa2e-0b16ea294a36.png)

1:1 pixel ratio of the scene.

**Comment by paluras** [comment]
Pirate
![pirate-static-hat-big](https://user-images.githubusercontent.com/98059798/195403739-aa3a5518-99bd-4496-a088-d65f5f7f0d15.gif)
![pirate-static-hat-](https://user-images.githubusercontent.com/98059798/195403741-5a824c94-400c-4b8d-a12b-e16078c0107d.gif)
![pirate-moving-hat](https://user-images.githubusercontent.com/98059798/195403743-07815656-38be-40e9-88e1-d692ccc289b8.gif)
![pirate-moving-hat-big](https://user-images.githubusercontent.com/98059798/195403745-1719b452-3789-4cfb-a997-24e17b2c9c92.gif)
![image](https://user-images.githubusercontent.com/98059798/195405327-a69edf5a-7c03-4b0a-bb36-7b98826901d1.png)

**Reply by On1el** [reply]
**Awesome!!**

**Comment by VadimasVP** [comment]
### Boat
![boat](https://user-images.githubusercontent.com/102822587/195417825-01b05e27-3208-4e0b-9d93-3ab35b8f3cbc.png)


![boat1](https://user-images.githubusercontent.com/102822587/195418288-40c93985-8186-426a-a30f-1f180f5e38de.png)

Pirate Boat
![pirateboat](https://user-images.githubusercontent.com/102822587/195419384-ef08d0b4-54dd-47d7-97c6-68d9bfd2032c.png)

Sunflower Boat
![sunflowerboat](https://user-images.githubusercontent.com/102822587/195419403-ba14655e-aa92-4248-b086-0fa12e0f3d4b.png)

Goblin Boat
![goblinboat](https://user-images.githubusercontent.com/102822587/195419488-d536b235-7908-4c28-bc50-446806acc300.png)

**Comment by TrevTweetz** [comment]
What do you need the size of the island icons to be?

**Comment by digital-farms** [comment]
## 🔥In game example
![Example-icons](https://user-images.githubusercontent.com/105886943/196138609-33ff09d7-23f7-4d30-bbb2-668fee63670f.png)

## 🏝️Main Island Icon
18x18
![main-icon](https://user-images.githubusercontent.com/105886943/195878211-e88b48b7-02a8-46f9-aba2-10af9c53435f.png)
180x180
![main-icon-1000](https://user-images.githubusercontent.com/105886943/195878263-bd4d6d0b-d9d8-4674-9019-e2b4bd91a782.png)

## 🌱Helios Island Icon
18x18
![helios-icon](https://user-images.githubusercontent.com/105886943/195859836-69b68531-db7a-41ad-aff7-2fe6fb923e17.png)
180x180
![helios-icon-1000](https://user-images.githubusercontent.com/105886943/195859875-43141aed-0b78-4e06-93db-6d0f90f4a17b.png)

## 🏦Goblin Retreat Icon
18x18
![retreat-icon](https://user-images.githubusercontent.com/105886943/195996754-614092a0-1d2a-4b11-a0fc-14718e2d8392.png)
180x180
![retreat-icon-1000](https://user-images.githubusercontent.com/105886943/195996767-a2742762-0d16-43f4-a449-8a590b1501f4.png)

## ❄️Snow Kingdom Icon
18x18
![snow-icon](https://user-images.githubusercontent.com/105886943/195860016-c26453da-8d32-4021-a021-d344caeb0f36.png)
180x180
![snow-icon-1000](https://user-images.githubusercontent.com/105886943/195860057-755684ef-3c33-48af-bbb6-34737c837db1.png)

**Comment by sfwhat** [comment]
**### Pirate**

![backggg](https://user-images.githubusercontent.com/95937808/195901326-786ef336-f27b-4d49-bd5f-a9bcea30401f.gif)
![pirate1](https://user-images.githubusercontent.com/95937808/195901331-470a97c8-0d8c-4a7e-a092-4eb3297985b8.gif)
![pirate](https://user-images.githubusercontent.com/95937808/195901334-e2d9b206-df5a-4c7e-8a67-0bb70cb5882d.gif)

---
SfWhat
Farm 6127

**Comment by On1el** [comment]
I really like the pirate that paluras drew, however I present my own version because I get pleasure from just drawing for my favorite project
I was inspired by Jack from Pirates of the Caribbean

![Джек500](https://user-images.githubusercontent.com/104461938/196005102-4e9f54ba-2320-4716-8093-6366851efcc1.gif)
![Джек11](https://user-images.githubusercontent.com/104461938/196005104-55e199bc-ab36-4cbf-9f52-ef926c84d5a6.gif)
![Джек5002](https://user-images.githubusercontent.com/104461938/196005608-f6b444e9-5325-4adc-b005-80ec89836aad.gif)

**Comment by FarmerKuenster** [comment]
Log raft to float away in!
version 1
![log-raft](https://user-images.githubusercontent.com/104322749/195964305-e2290cbb-3991-4aa3-bb32-e5cf8563d1ea.png)

![boatlog-raft-in-game](https://user-images.githubusercontent.com/104322749/195964231-804c3d00-1906-4872-bb96-04cd6a840866.png)

version 2

![log-raft-version2](https://user-images.githubusercontent.com/104322749/196001749-7d0f7817-962c-483b-8b46-7b94fca45ea3.png)


![boatlog-raft-version2-in-game](https://user-images.githubusercontent.com/104322749/196001756-37f46ddb-e07d-4802-95eb-356ef784dfab.png)



discord: farmerkunester #1626
farm ID #32391

**Comment by VyroxXx** [comment]
Farm ID: 31562

Main Island Icon:
![mainisland100](https://user-images.githubusercontent.com/114549921/196020297-6821a036-16d4-47a9-be8e-1540b495b5e7.png)

20x20
![mainisland20](https://user-images.githubusercontent.com/114549921/196020302-36a7af22-80f9-41f2-bcc7-d45b1029a1c2.png)

10x10
![main10](https://user-images.githubusercontent.com/114549921/196530839-6bb82e89-c96d-44b0-b141-b5d9d964b10d.png)

Snow Kingdom Icon:
![neve100](https://user-images.githubusercontent.com/114549921/196014008-b7db59d1-2363-4080-9867-6419bc95ac25.png)

20x20
![neve20](https://user-images.githubusercontent.com/114549921/196014247-87bf1cc3-8dcd-4e85-9a95-facbe4332b0b.png)

10x10
![neve10](https://user-images.githubusercontent.com/114549921/196530905-48a1d08c-ae0b-4c0b-b083-f90ea84473b1.png)

Helios Icon: (where we have the Grub, Potion and Magic Seeds shops)
![Helios100](https://user-images.githubusercontent.com/114549921/196018598-df846f20-20d3-4a18-8ee0-3bce50b06624.png)

20x20
![Helios20](https://user-images.githubusercontent.com/114549921/196018603-a60ddad4-b4a0-459d-ae14-7299123ebf6c.png)

10x10
![helios10](https://user-images.githubusercontent.com/114549921/196530972-41d8e0ce-b3e0-4bd3-84a2-51ace1694ec1.png)

poison island Icon:
![Poison100](https://user-images.githubusercontent.com/114549921/196018612-9542dc55-7918-4000-8e79-578585613a74.png)

20x20
![Poison20](https://user-images.githubusercontent.com/114549921/196018615-0dc54641-06e7-4fac-85ad-c14eacd2676c.png)

10x10
![poison10](https://user-images.githubusercontent.com/114549921/196531046-224b87bd-850a-4b72-b7b6-781b053976cb.png)

Goblin Retreat Icon:
![gob100](https://user-images.githubusercontent.com/114549921/196048764-97d52927-42f0-47ee-a9c8-6151a59344e0.png)

20x20
![gob20](https://user-images.githubusercontent.com/114549921/196048776-11b64dba-db71-4233-b4a2-e2f266186ed5.png)

10x10
![goblin10](https://user-images.githubusercontent.com/114549921/196531112-e53ec3ea-cf93-4b9d-a8aa-1f809e2988fc.png)

**Reply by B3atrizR3ach** [reply]
Goos ideas!

**Comment by FarmerKuenster** [comment]
Island Icon Ideas!
(will add the other badges as I complete them)

Goblin Retreat option 1
![island-icon-goblin-retreat-1](https://user-images.githubusercontent.com/104322749/196500780-cda9c3da-f657-4536-94a7-2178e7f125d0.png)

![island-icon-goblin-retreat-1-LARGE](https://user-images.githubusercontent.com/104322749/196501279-36373a2d-1f39-4e72-8cac-2ec7ba5a06d5.png)


Goblin Retreat option 2
![island-icon-goblin-retreat-2](https://user-images.githubusercontent.com/104322749/196501331-e7e6492b-95ec-465e-8dc2-0f670a96f4f3.png)

![island-icon-goblin-retreat-2-LARGE](https://user-images.githubusercontent.com/104322749/196501347-e117d158-7f4e-4738-b031-e91b1a032446.png)

Helios (where we have the Grub, Potion and Magic Seeds shops)
![island-icon-Helios](https://user-images.githubusercontent.com/104322749/196503793-ea9ff59e-e98b-41fb-8bbd-5e05a9565347.png)

![island-icon-Helios-LARGE](https://user-images.githubusercontent.com/104322749/196503806-1096ca30-3df6-47b2-a8fd-1d8992558e27.png)

Main Island
![island-icon-main-island](https://user-images.githubusercontent.com/104322749/196573734-a32c10c0-715d-4fc9-9fec-f7aae5e892f0.png)


![island-icon-main-island-LARGE](https://user-images.githubusercontent.com/104322749/196573583-ec5e7c26-d4d0-4101-9010-dac2814cc6b4.png)

Snow Kingdom
![island-icon-snow-kingdom](https://user-images.githubusercontent.com/104322749/196577535-2c757a1d-aeb4-4f70-8397-b6b78763bd7f.png)


![island-icon-snow-kingdom-LARGE](https://user-images.githubusercontent.com/104322749/196577543-5ab2db14-77a4-4a4a-be76-d56026de9b15.png)


discord: farmerkunester #1626
farm ID #32391

**Comment by digital-farms** [comment]
## 🎄Xmas reindeer
![olen-snow_kingdom](https://user-images.githubusercontent.com/105886943/207833827-6f75fb33-9099-45d7-b3bd-1bd04a272a61.gif)
![olen-xmas](https://user-images.githubusercontent.com/105886943/207833878-cc47df0b-a00a-4242-9887-d42e166fd06c.gif)
